sources/Licenses/Licenses.yml

```
@@ -0,0 +1,52 @@
+---
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+action:
+  name: replace
+swap:
+  "[0O]BSD": 0BSD
+  BSD: BSD
+  BSD[ -]?1: BSD-1
+  BSD[ -]?2: BSD-2
+  BSD[ -]?3: BSD-3
+  CPL: CPL
+  CPL[ -]?1(?:\.0)?: CPL 1.0
+  EPL: EPL
+  EPL[ -]?1(?:\.0)?: EPL 1.0
+  EPL[ -]?2(?:\.0)?: EPL 2.0
+  EUPL: EUPL
+  EUPL[ -]?v1(?:\.2)?: EUPL v1.2
+  ISC: ISC
+  MIT: MIT
+  MirOS: MirOS
+  MPL: MPL
+  MPL[ -]?2(?:\.0)?: MPL 2.0
+  OFL: OFL
+  WTFPL: WTFPL
+  Zlib: Zlib
```

```suggestion
  zlib: zlib
```

Lowercase, no?

https://en.m.wikipedia.org/wiki/Zlib_License

---

I saw it written ``Zlib`` elsewhere, but ``zlib`` does indeed seem to be the preference.